### DISCUSSION TITLE
implement the ListBoxSection component

### DISCUSSION THREAD

**Discussion by birchrust** [adobe/react-spectrum#4657] [Q&A] [answered]
Property 'rendered' and 'key' does not exist on type 'AriaListBoxSectionProps'
Is the type of ListBoxSectionProps wrong?
![image](https://github.com/adobe/react-spectrum/assets/121175143/64884b1c-e398-49ee-b9fc-d4c6a756b354)

**Comment by LFDanLu** [comment, marked as answer]
Those props come from the Node type def: https://github.com/adobe/react-spectrum/blob/6a858e3663aea1fb2b2a2ca5d2b3fb4ddeae99e4/packages/%40react-types/shared/src/collections.d.ts#L164-L177. which our own ListBoxSection makes use of here: https://github.com/adobe/react-spectrum/blob/6a858e3663aea1fb2b2a2ca5d2b3fb4ddeae99e4/packages/%40react-spectrum/listbox/src/ListBoxSection.tsx#L25

**Reply by birchrust** [reply]
thanks
